Spigot-Server-Patches/0645-fix-converting-txt-to-json-file.patch

```diff
@@ -0,0 +1,62 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Mon, 4 Jan 2021 19:49:15 -0800
+Subject: [PATCH] fix converting txt to json file
+
+
+diff --git a/src/main/java/net/minecraft/server/DedicatedPlayerList.java b/src/main/java/net/minecraft/server/DedicatedPlayerList.java
+index a8cb602be410fb269f4d7e19e40ee07a84ea86e5..73b1022e192d2cc4a953a784408ef909a3192c59 100644
+--- a/src/main/java/net/minecraft/server/DedicatedPlayerList.java
++++ b/src/main/java/net/minecraft/server/DedicatedPlayerList.java
+@@ -15,6 +15,11 @@ public class DedicatedPlayerList extends PlayerList {
+ 
+         this.a(dedicatedserverproperties.viewDistance);
+         super.setHasWhitelist((Boolean) dedicatedserverproperties.whiteList.get());
++        // Paper start - moved from constructor
++    }
++    @Override
++    public void loadAndSaveFiles() {
++        // Paper end
+         this.y();
+         this.w();
+         this.x();
+diff --git a/src/main/java/net/minecraft/server/DedicatedServer.java b/src/main/java/net/minecraft/server/DedicatedServer.java
+index 5504facd2e453238caa71d98743be5416d4dd4fe..5a16ebcdcf9a0e5432126fc912ac5ee52283e682 100644
+--- a/src/main/java/net/minecraft/server/DedicatedServer.java
++++ b/src/main/java/net/minecraft/server/DedicatedServer.java
+@@ -159,6 +159,12 @@ public class DedicatedServer extends MinecraftServer implements IMinecraftServer
+         org.spigotmc.SpigotConfig.init((java.io.File) options.valueOf("spigot-settings"));
+         org.spigotmc.SpigotConfig.registerCommands();
+         // Spigot end
++        // Paper start - moved up to right after PlayerList creation but before file load/save
++        if (this.convertNames()) {
++            this.getUserCache().b(false); // Paper
```

Comment:
this should use an obf helper, else it might change to something unrelated since the other change is too far away

Reply:
done